Makefile

```diff
@@ -0,0 +1,66 @@
+CHS_ENV_HOME ?= $(HOME)/.chs_env
+TESTS        ?= ./...
+
+bin          := elasticsearch-data-loader
+cb           := companybindex
+chs_env      := $(CHS_ENV_HOME)/global_env
+source_env   := for chs_env in $(chs_envs); do test -f $$chs_env && . $$chs_env; done
+xunit_output := test.xml
+lint_output  := lint.txt
+
+commit       := $(shell git rev-parse --short HEAD)
+tag          := $(shell git tag -l 'v*-rc*' --points-at HEAD)
+version      := $(shell if [[ -n "$(tag)" ]]; then echo $(tag) | sed 's/^v//'; else echo $(commit); fi)
+
+.EXPORT_ALL_VARIABLES:
+GO111MODULE = on
+
+.PHONY: all
+all: clean build
+
+.PHONY: fmt
+fmt:
+	go fmt ./...
+
+.PHONY: build
+build: fmt
+	go build
+
+.PHONY: test
+test: test-unit test-integration
+
+.PHONY: test-unit
+test-unit: test-deps
+	@set -a; go test $(TESTS) -run 'Unit'
+
+.PHONY: test-integration
+test-integration:
+	$(source_env); go test $(TESTS) -run 'Integration'
+
+.PHONY: clean
+clean:
+	rm -f $(bin)
+	rm -f *.zip
+	rm -rf build-*
+
+.PHONY: package
+package:
+	$(eval tmpdir:=$(shell mktemp -d build-XXXXXXXXXX))
+	cp -r ./$(cb) $(tmpdir)/$(cb)
```

Comment:
companybindex doesn't exist on the master branch

Reply:
Shall I comment it out for now as we know it will be required very soon?

Reply:
Commented out code is usually bad practise and can cause confusion whether it is needed or not. I would advise removing for now. It'll be in the commit history when you need it.

Reply:
Rightio